### PR TITLE
[ecosystem] Fix Register / Sign in Dialog + General UI fixes

### PR DESCRIPTION
**Pull request by moonclavedev**
## Motivation

We needed to fix display issues (mostly for mobile) to make the Register / Sign In modal usable. 
Make UI fixes for Aptos Incentivized Testnet.

### Current:

Messages Image(104891942)
image

### Proposed:

image
image

### **Global**

- Update page `<title>` - change from "Community Platform" to "Aptos Community Platform" 
- Match brand with exact hex for green including an updated palette with complete family of shades as a new color object titled: "**surf**".
- Adjust "Success" Alert color scheme to match branding instead of the blue
- Change `<a>` underline styling for better readability
- Border Radius updated to match aptoslabs.com and [explorer](https://explorer.devnet.aptos.dev/)
- Center text within Footer (mobile only)

### **Index**

- Set first sentence in into paragraph to `<h2>` and increase font size for better text heirarchy.
- Change language to match what's being used on https://aptoslabs.com: **Register** / **Sign In** (as opposed to Sign up / Log in)
- Change Sign in button as hollow / outlined to differentiate from Register

### **Register / Sign in Dialog**

- Remove white background from modal to stay consistent with dark mode theme styles
- Adjust header styling
- Integrate new buttons for SSO services to match their respective branding (and latest version of Github Octocat logo) including name, which may prove beneficial from a compliance perspective in addition to providing additional trust to the user from a visual perspective.
- Enabled modal to be closed when user clicks anywhere outside of the dialog as opposed to only being able to click the "x"

### **Form**

- Validate Node / Submit container is now `position:sticky` (Medium to Large screens) and will no longer be out of view when user has scrolled the page as they complete the fields, eliminating need to scroll back up the page to find the submit button. Set container full width on small screens.
- Convert `<form>` to flex, and set gap to prevent Validate Node / Submit container from overlapping on Medium screens

### CODE REVIEW

File: ecosystem/platform/server/config/tailwind.config.js [hunk 1]
```diff
@@ -23,6 +23,18 @@ module.exports = {
           '400': '#1bd7a4',
           '500': '#187e65',
         },
+        surf: {
```
Comment (zacharydenton) [inline ecosystem/platform/server/config/tailwind.config.js:26]: Let's delete the changes to `teal` in this file now that we have the full palette.

**Review by zacharydenton — APPROVED** [review]
LGTM

**Comment by davidiw** [comment]
I like it!

**Comment by davidiw** [comment]
@moonclavedev is this PR still valid?

**Pull request closed without merging**